### PR TITLE
Css tidy up

### PR DESCRIPTION
**Pull request by rupertbates**
This PR addresses some of the feedback from the ['New Landing Page for Supporter in the UK'](https://github.com/guardian/membership-frontend/pull/1362) PR.

I've left the hr elements in place because I couldn't find a way to get the 80px width using ::after but I've moved them inside the li elements to address 'element not allowed here' warnings.

@svillafe @SiAdcock could you take a look please?

### CODE REVIEW

**Comment by SiAdcock** [comment]
Looks good. Shame about the `hr` elements, but it's much better now they're inside the `li` elements 😄  👍 

Did you see [this SO answer](http://stackoverflow.com/questions/8835142/any-way-to-declare-a-size-partial-border-to-a-box#8837382)?

File: frontend/assets/stylesheets/components/_elevated-button.scss [hunk 1]
```diff
@@ -39,9 +39,13 @@
...
-    text-align: center;  
+    border-radius: 50%;
+    text-align: center;
     > svg {
         margin-top: 7px; /*To vertically align the button's logo*/
     }
 }
+
+.elevated-button__icon__border svg {
+    fill : #FFFFFF;
```
Comment (SiAdcock) [inline frontend/assets/stylesheets/components/_elevated-button.scss]: super-nitpick: can this be lowercase for consistency?

**Comment by svillafe** [comment]
Looks nice, I solved the little details that @SiAdcock found and ship it 🚀

**Comment by rupertbates** [comment]
@SiAdcock - I had another go at removing the hr elements using the SO answer you linked and managed to do it so thanks for that. Also colour is now lower case.

**Comment by SiAdcock** [comment]
Awesome work @rupertbates  👍

**Pull request merged**